src/main/java/makePost/use_case/MakePostDataAccessInterface.java

```diff
@@ -0,0 +1,22 @@
+package makePost.use_case;
```

Comment:
Also, could we rename this to `MakePostDsGateway`? We are using the `DsGateway` suffix to denote the data access interface.